index.html

```diff
@@ -188,5 +188,23 @@ <h2 class="presentation2">About <br> myself <br></h2>
 </div>
 </section>
 <!-- about myself finish-->
+  <!-- contact form start -->
+<section class="sec3">
+      <div class="contact">
+          <div class="contact_text">
+          <h2 class="contactme">Contact me</h2>
+          <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+          </div>
+          <form>
+              <input type="text" name="name" placeholder="Yeremias" required>
```

Comment:
- [ ] You should add validation to limit the maximum characters for full name input field to `30 characters `as per the project requirement.